app/components/request_form/request_form.css

```diff
@@ -10,6 +10,20 @@
   top: calc(3.5 * var(--spacing-unit));
 }
 
+.RequestForm-messageTextAreaToolbar {
+  width: 100%;
+  border: 2px solid transparent;
+  border-radius: 5px;
+  padding: 6px;
+  background-color: var(--color-gray-light);
+  margin: var(--spacing-unit-s) 0;
+}
+
+.RequestForm-insertPlaceholderButton {
+  background-color: #fff;
+  font-size: 1rem;
```

Comment:
I don't like the idea of text being smaller than `1rem/16px` because this is generally accepted as the minimum font-size for `a11y`. See [Minimum font size?](https://accessibleweb.com/question-answer/minimum-font-size/)

> While there is no official minimum font size for the web, it is generally agreed upon that 16px for body text is a good starting point.

I personally would like to scale up the font-size to meet this suggestion, but not sure how others feel.

I did make the font-size by adding CSS to decrease the font-size from our normal buttons.
One thing we talked about potentially doing is using an icon with a tooltip that contains the text.

Reply:
Alright, I see the accessibility point here. However, we currently have two system font sizes that are smaller than `1rem`:
`--font-size-s: 0.85em; 
--font-size-xs: 0.75em;`

The help text to whose font size I proposed matching the button font size e.g. has `--font-size-s`. I am definitely open to discuss increasing our font sizes to a minimum of `1rem` (I don't see any down sides), but I would like to do this in a consistent effort to avoid mismatching proportions.

I therefore adapted my suggestion to `font-size: var(--font-size-s)` and would propose also opening an issue about increasing our system font sizes to a minimum of `1rem`(luckily with the font-size variables that would be easy to change).

Reply:
Added an issue: https://github.com/tactilenews/100eyes/issues/1444

Reply:
I think that can be a nice compromise, though I am a bit hesitant cause in my experience there almost always is a lack of interest in prioritizing this kind of change.

Reply:
Let's start a discussion on the issue then. (I think its cleaner and more impactful to have this discussion for the whole application.)